Console release notes 2022-09-02 (#142)
neondatabase/website: content/docs/release-notes.md

---
title: Release Notes
---
## 2022-09-02 Console release

### What's new

> - UI: Pass OAuth app name to the client

Reviewer: let's stick to lowercasing here `Pass` -> `pass`.

> - UI: Pass OAuth app name to the client

Reviewer 2: ```suggestion
- UI: pass OAuth app name to the client
```

### Bug fixes

> - UI: fix title on the Sing in page

Reviewer: `Sing` -> `sign`?

> - UI: fix title on the Sing in page

Reviewer 2: ```suggestion
- UI: fix title on the 'Sign in' page.
```

> - UI: fix postgres version.

Reviewer 2: ```suggestion
- UI: change Postgres version on dashboard to 14.5.
```

> 

Reviewer 2: ```suggestion

- Control plane: fix authentication of concurrent proxy connections to the idle compute node.

```
